src/Controller/IgnoreOriginInstanceRuleController.php

```diff
@@ -48,6 +50,7 @@ public function indexAction(IgnoreOriginInstanceRuleRepository $repository)
      * Creates a new ignore origin instance rule entity.
      *
      * @Route("/new", name="ignore_origin_instance_rules_new", methods={"GET", "POST"})
+     * @IsGranted("CREATE_IGNORE_ORIGIN_INSTANCE_RULES")
```

Comment:
To be consistent with `CREATE_USER`

```suggestion
     * @IsGranted("CREATE_IGNORE_ORIGIN_INSTANCE_RULE")
```

Reply:
@j0k3r I'd tend to prefer updating to `CREATE_USERS` then, what do you think?

Reply:
Yes ok